### PR TITLE
Created my first Goodie

### PR DESCRIPTION
**Pull request by marcantonio**
**What does your Instant Answer do?**
Tells everyone how awesome I am.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
That not enough people know that I'm awesome.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Me.

**Why did you choose this data source?**
I'm authoritative on the subject.

**Are there any other alternative (better) data sources?**
Perhaps my wife, but unlikely.

**What are some example queries that trigger this Instant Answer?**
duckduckhack marcantonio

**Where did you hear about DuckDuckHack? (For first time contributors)**
Somewhere on the Internet.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-02-26 at 9 25 29 pm

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [X] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [X] Google Chrome
        - [X] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

File: t/IsAwesome/marcantonio.t [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+use Test::More;
+use DDG::Test::Goodie;
+
+zci answer_type => "is_awesome_marcantonio";
+zci is_cached   => 1;
+
+ddg_goodie_test(
+    [qw(
+       DDG::Goodie::IsAwesome::marcantonio
+	     )],
```
Comment (mintsoft) [inline t/IsAwesome/marcantonio.t]: Can you fix the indentation here? Should be 4 spaces :

**Comment by mintsoft** [comment]
Congratulations on your first goodie @marcantonio , there's just the small indentation issue to fix then it'll be ready for merging!

**Comment by marcantonio** [comment]
There you go.  Thanks!

**Comment by mintsoft** [comment]
Thanks for the quick fix @marcantonio. Congratulations on your first goodie! :+1:

**Comment by moollaza** [comment]
@marcantonio congrats! It's live: https://duckduckgo.com/?q=duckduckhack+marcantonio&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**